EventsExpress/Controllers/NotificationTemplateController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.IServices;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EventsExpress.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class NotificationTemplateController : Controller
+    {
+        private readonly INotificationTemplateService _notificationTemplateService;
+
+        public NotificationTemplateController(INotificationTemplateService notificationTemplateService)
+        {
+            _notificationTemplateService = notificationTemplateService;
+        }
+
+        [HttpPost("Add")]
+        public async Task<ActionResult> Add(NotificationTemplateDTO notificationTemplateDto)
+        {
+            if (!ModelState.IsValid)
```

Comment:
We don't do the `ModelState` validation in controllers. It is automatically done in the binder.